builder_test.go

```diff
@@ -630,6 +630,21 @@ func TestBuilder(t *testing.T) {
 				},
 			},
 		},
+		{
+			Dockerfile: "dockerclient/testdata/Dockerfile.healthcheckdefaults",
+			From:       "debian",
+			Config: docker.Config{
+				Image: "debian",
+				Cmd:   []string{"/bin/sh", "-c", "/app/main.sh"},
+				Healthcheck: &docker.HealthConfig{
+					StartPeriod: 0 * time.Second,
+					Interval:    30 * time.Second,
+					Timeout:     30 * time.Second,
+					Retries:     3,
+					Test:        []string{"CMD-SHELL", "/app/check.sh --quiet"},
+				},
+			},
+		},
```

Comment:
I'm probably overthinking, but I just wonder if we're starting to brush up against DockerHub rate limits.  No need to adjust,  just thought I'd put that thought out there.

Reply:
We tend to use one of a handful of base images for various tests, and the docker daemon hangs on to them between tests, but you're not wrong to be concerned that it will become a problem at some point.